<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRelationToProgrammer</class>
 <widget class="QDialog" name="AddRelationToProgrammer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Relations</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>19</y>
     <width>361</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_the_programmer">
      <property name="text">
       <string>&lt;h2&gt;Add relations for Programmer&lt;/h2&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="input_search_computers"/>
    </item>
    <item>
     <widget class="QListWidget" name="list_computers_to_relate"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="button_cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_finish">
        <property name="text">
         <string>Finish</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
